### PR TITLE
Fix CP configuration

### PR DESCRIPTION
**Pull request by dabrt**
| Question      | Answer |
| ------------- | ---
| JIRA Ticket   | none |
| Versions      | 4.5 |
| Edition       |  Commerce |

Add a mention that one must have all potential currencies configured before CP users can review their orders.

#### Checklist

- [x] Text renders correctly
- [x] Text has been checked with vale

### CODE REVIEW

File: docs/customer_management/cp_configuration.md [hunk 1]
```diff
@@ -89,3 +89,8 @@ ibexa:
                         match:
                             Identifier\ContentType: confirmation_page
 ```
+
+# Order management
+
+Reviewing pending and past orders in Customer Portal requires that you configure all currencies that any of the customers may use under the `ibexa.system.default.product_catalog.currencies` key.
```
Comment (konradoboza) [inline docs/customer_management/cp_configuration.md:95]: Feel free to rephrase it if needed, we need to emphasize that first configured currency will be the one that the list is showing:
```suggestion
Reviewing pending and past orders in Customer Portal requires that you configure all currencies that any of the customers may use under the `ibexa.system.default.product_catalog.currencies` key. The first currency from the list will be taken for filtering orders list as well as for calculating "Average order" and "Total amount".
```

**Review by konradoboza — APPROVED** [review]

File: docs/customer_management/cp_configuration.md [hunk 2]
```diff
@@ -89,3 +89,9 @@ ibexa:
...
```
Comment (adamwojs) [inline docs/customer_management/cp_configuration.md:95]: Why currencies list needs to be defined in `default` scope ?

Comment (konradoboza) [reply docs/customer_management/cp_configuration.md:95]: Good catch, it actually doesn't. We can change it to sth like:
`ibexa.system.<your_siteaccess>.product_catalog.currencies`.

**Review by julitafalcondusza — APPROVED** [review]

**Review by adamwojs — APPROVED** [review]

File: docs/customer_management/cp_configuration.md [hunk 3]
```diff
@@ -89,3 +89,9 @@ ibexa:
                         match:
                             Identifier\ContentType: confirmation_page
 ```
+
+# Order management
+
+Reviewing pending and past orders in Customer Portal requires that you configure all currencies that any of the customers may use under the `ibexa.system.<siteaccess_name>.product_catalog.currencies` key. 
+The first currency from the list is then used for filtering the orders list and calculating the **Average order** and **Total amount** values.
+For more information, see [Enable purchasing products](https://doc.ibexa.co/en/latest/pim/enable_purchasing_products/).
```
Comment (adriendupuis) [inline docs/customer_management/cp_configuration.md:97]: It shouldn't have been passed as an external link.
`[Enable purchasing products](enable_purchasing_products.md)` would have been better.

**Pull request merged**